src/main/java/io/odpf/kafkaclient/consumer/NullTopicException.java

```diff
@@ -1,4 +1,21 @@
 package io.odpf.kafkaclient.consumer;
 
-public class NullTopicException extends RuntimeException{
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+public class NullTopicException extends RuntimeException {
```

Comment:
Same as other Exception.

Reply:
Deleted the override and implemented constructor for exceptions